tests/debugger/test_debugger.py

```diff
@@ -76,23 +78,95 @@ def check_info_endpoint():
 @missing_feature(
     context.library == "java" and context.weblog_variant not in ["spring-boot", "uds-spring-boot"],
     reason="not supported",
 )
+@irrelevant(library="golang")
+@irrelevant(library="python")
+@irrelevant(library="ruby")
+@irrelevant(library="php")
+@irrelevant(library="nodejs")
+@irrelevant(library="cpp")
 @scenarios.debugger_method_probes_status
 class Test_Debugger_Method_Probe_Statuses:
     def test_method_probe_status(self):
-        check_probe_statuses(METHOD_PROBES)
+        expected_data = [
+            "logProbe-received",
+            "logProbe-installed",
+            "metricProbe-received",
+            "metricProbe-installed",
+            "spanProbe-received",
+            "spanProbe-installed",
+            "spanDecorationProbe-received",
+            "spanDecorationProbe-installed",
+        ]
+        validate_data(expected_data, [])
 
 
-@released(java="1.19.3", php_appsec="?", python="?")
 @missing_feature(
     context.library == "java" and context.weblog_variant not in ["spring-boot", "uds-spring-boot"],
     reason="not supported",
 )
+@irrelevant(library="golang")
+@irrelevant(library="python")
+@irrelevant(library="ruby")
+@irrelevant(library="php")
+@irrelevant(library="nodejs")
+@irrelevant(library="cpp")
 @scenarios.debugger_line_probes_status
 class Test_Debugger_Line_Probe_Statuses:
     def test_line_probe_status(self):
-        check_probe_statuses(LINE_PROBES)
+        expected_data = ["logProbe-installed", "metricProbe-installed", "spanDecorationProbe-installed"]
+        validate_data(expected_data, [])
+
+
+@missing_feature(
+    context.library == "java" and context.weblog_variant not in ["spring-boot", "uds-spring-boot"],
+    reason="not supported",
+)
+@irrelevant(library="golang")
+@irrelevant(library="python")
+@irrelevant(library="ruby")
+@irrelevant(library="php")
+@irrelevant(library="nodejs")
+@irrelevant(library="cpp")
+@scenarios.debugger_method_probes_snapshot
+class Test_Debugger_Method_Probe_Snaphots:
+    remote_config_is_sent = False
+    probe_installed = False
+    logProbeResponse = None
+
+    def setup_method_probe_snaphots(self):
+        def wait_for_remote_config(data):
+            if data["path"] == "/v0.7/config":
+                if "client_configs" in data.get("response", {}).get("content", {}):
+                    self.remote_config_is_sent = True
+                    return True
+            return False
+
+        def wait_for_probe(data):
+            if data["path"] == "/api/v2/logs":
+                contents = data.get("request", {}).get("content", {})
+                for content in contents:
+                    debuggerData = content["debugger"]
+                    if "diagnostics" in debuggerData:
+                        if debuggerData["diagnostics"]["status"] == "INSTALLED":
+                            self.probe_installed = True
+                            return True
+            return False
+
+        print(self.remote_config_is_sent)
```

Comment:
Could you remove the `print` ?

Reply:
Removed here bec964f9db45bba269a8b858f4d35a2bbb11fb9b